types/node/index.d.ts

```diff
@@ -6818,3 +6819,205 @@ declare module "http2" {
     export function connect(authority: string | url.URL, listener?: (session: ClientHttp2Session, socket: net.Socket | tls.TLSSocket) => void): ClientHttp2Session;
     export function connect(authority: string | url.URL, options?: ClientSessionOptions | SecureClientSessionOptions, listener?: (session: ClientHttp2Session, socket: net.Socket | tls.TLSSocket) => void): ClientHttp2Session;
 }
+
+declare module "perf_hooks" {
+	interface PerformanceEntry {
```

Comment:
Are there really all properties optional? I would assume that only `kind` is optional.